docs/Testing-Transmission.md

```diff
@@ -6,11 +6,20 @@ Download the desired "from-tarball" Artifact at the bottom of the page.
 
 ## How to open a CI build on macOS ##
 
-You need to mark Transmission.app both as executable and not quarantined:
+You need to mark Transmission.app both as executable and not quarantined using the Terminal application (in Applications/Utilities):
 
 1. `chmod +x Transmission.app/Contents/MacOS/Transmission`
+
+   If you're unfamiliar with using the Terminal application, the following steps might be of help.
+
+   1. Open the Terminal application and type: chmod +x (and add a space after the x).
+   2. Right-click or Ctrl-click the Transmission application and select 'Show Package Contents'.
+   3. Navigate to the 'MacOS' directory; drag and drop 'Transmission' on to the Terminal window you had opened in step 1 and press Return. 
```

Comment:
This is over complicated and unnecessary.

Let's just add "_using the Terminal application, go to the folder containing Transmission.app and copy-paste those commands_".